R/hugo.R

```diff
@@ -179,15 +178,16 @@ install_theme = function(
       'and at least take a look at the config file config.toml of the example site, ',
       'because not all Hugo themes work with any config files.'
     )
-    newdir = gsub(sprintf('-%s$', branch), '', zipdir)
+    newdir = gsub(tmpdir, ".", zipdir)
+    newdir = gsub(sprintf('-%s$', branch), '', newdir)
```

Comment:
@yihui Should I add an `if (!theme_is_url)` here so that it doesn't even try to change the filename if the zip file is given? The alternative would be to add some logic around the branch regex and then hope we get lucky at this stage ... so seems to me to be better to skip this if `theme_is_url`. I didn't do this because I was trying not to add a ton of logic and branching paths, but it's been nagging at me.

Reply:
Conditioning on `theme_is_url` makes sense to me.